api/v1beta2/foundationdbcluster_types.go

```diff
@@ -2189,6 +2216,34 @@ func (cluster *FoundationDBCluster) RemoveProcessGroupsFromCrashLoopList(process
 	cluster.Spec.Buggify.CrashLoop = cluster.Spec.Buggify.CrashLoop[:idx]
 }
 
+// RemoveProcessGroupsFromCrashLoopContainerList removes the provided process group IDs from the crash-loop container list.
+func (cluster *FoundationDBCluster) RemoveProcessGroupsFromCrashLoopContainerList(processGroupIDs []ProcessGroupID, containerName string) error {
+	processGroupIDsToRemove := make(map[ProcessGroupID]None)
+	for _, processGroupID := range processGroupIDs {
+		processGroupIDsToRemove[processGroupID] = None{}
+	}
+
+	crashLoopIdx := 0
+	for _, crashLoopContainerObj := range cluster.Spec.Buggify.CrashLoopContainers {
+		if containerName != crashLoopContainerObj.ContainerName {
+			crashLoopIdx++
+			continue
+		}
+		newTargets := make([]ProcessGroupID, 0)
+		for _, processGroupID := range crashLoopContainerObj.Targets {
+			if _, ok := processGroupIDsToRemove[processGroupID]; ok {
+				continue
+			}
+			newTargets = append(newTargets, processGroupID)
+		}
+		crashLoopContainerObj.Targets = newTargets
+		cluster.Spec.Buggify.CrashLoopContainers[crashLoopIdx] = crashLoopContainerObj
+
+		return nil
+	}
+	return fmt.Errorf("container: %s not found", containerName)
```

Comment:
Not sure if we really want to throw an error here, do you think that make sense?